src/client/pages/Profile.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+import Navbar from '../components/Navbar';
+import Searchbar from '../components/Searchbar';
+
+
+import Button from '../components/Button';
+
+import '../style/Profile.css';
+
+function Profile() {
+  const location = useLocation();
+  const history = useHistory();
+  const [username, setUsername] = useState('Username');
+  const [email, setEmail] = useState('username@gmail.com');
+  const [about, setAbout] = useState('Ernest Miller Hemingway (July 21, 1899 – July 2, 1961) was an American journalist, novelist, short-story writer, and sportsman. His economical and understated style—which he termed the iceberg theory—had a strong influence on 20th-century fiction, while his adventurous lifestyle and his public image brought him admiration from later generations. Hemingway produced most of his work between the mid-1920s and the mid-1950s, and he won the Nobel Prize in Literature in 1954. He published seven novels, six short-story collections, and two nonfiction works. Three of his novels, four short-story collections, and three nonfiction works were published posthumously. Many of his works are considered classics of American literature. ');
+
+  const handleEdit = () => {
+    const request = {
+      method: 'POST',
+      mode: 'same-origin',
+      headers: { 'Content-Type': 'application/json' },
+      body: {
+        username: 'test',
+        password: 'test',
+        email: 'test'
+      }
+    };
+    fetch('http://localhost:5000/api/register', request);
+    console.log('registration successful');
+  };
+
+  return (
+    <body>
```

Comment:
Using `body` tag inside another `body` tag is not recommended. And the `main` tag here adds additional nesting level. Maybe we can use some simplification here.

Reply:
simplified tag structure: removed unnecessary <body> amd <main> tags